test/OpenTelemetry.Tests/Trace/TracerProviderBuilderExtensionsTest.cs

```diff
@@ -133,6 +133,75 @@ public void ServiceLifecycleAvailableToSDKBuilderTest()
         Assert.True(myInstrumentation.Disposed);
     }
 
+    [Fact]
+    public void AddProcessorTest()
```

Comment:
As there is a lot of discussion about not breaking the existing behavior, should we have one test with many processors, without PipelineWeight, to validate it?